google/cloud/security/scanner/scanners/fw_rules_scanner.py

```diff
@@ -23,6 +23,7 @@
 
 from google.cloud.security.common.data_access import csv_writer
 from google.cloud.security.scanner.audit import fw_rules_engine
+from google.cloud.security.common.gcp_type import resource_util
```

Comment:
pylint might complain that this isn't alphasorted?

Reply:
Haha looks like pylint didn't complain. But could you alphasort these imports?

from g.c.s.common.data_access import ...
from g.c.s.common.gcp_type import ...
from g.c.s.scanner.audit import ...
from g.c.s.scanner.scanners import ...